Formula/protoc-gen-go.rb

```diff
@@ -18,7 +22,7 @@ class ProtocGenGo < Formula
   depends_on "protobuf"
 
   def install
-    system "go", "build", *std_go_args, "-ldflags", "-s -w", "./protoc-gen-go"
+    system "go", "build", *std_go_args, "-ldflags", "-s -w", "./cmd/protoc-gen-go"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/protoc-gen-go"
```
Let's use the updated syntax for passing `ldflags`.

Reply:
Done.